rd-net/Lifetimes/Diagnostics/Mode.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace JetBrains.Diagnostics;
+
+public static class Mode
+{
+  public static readonly bool Assertion = AppDomain.CurrentDomain.GetData("JET_MODE_ASSERT") as bool? == true;
```

Comment:
I'd rather like to see it as `IsAssertion` in code, so that it were clear it's a flag and not an assertion object with some functionality.

Reply:
ok no objections